Accessibility

IeuanWalker opened the pull request:
The platform specific changes basically tell the OS that the GestureFrame is a button. So the OS will then treat it as a button from an accessibility POV.

New property added called `AccessibleName`, this allows the user to manually set the accessible name, if not set it will default to the `Text` property value

_Haven't done UWP, sorry_

IeuanWalker committed:
Accessibility

robinmanuelthiel commented:
Hi @IeuanWalker, thanks a lot for that addition! Will review ASAP. One question: Did you actually change anything in the `.xaml` and `.csproj` and `.sln` files? Looks like you are using a different formatter and there are A LOT of code changes in these files. Would be easier to review, if I knew, what ACTUALLY changed!

IeuanWalker commented:
No prob, ye sorry about that.

I never touched the  '.csproj' and '.sl' files. 
The only `.xaml` file i changed was the `FlexDemoPage` on line 83 i added `AccessibleName="Light bulb"` to test the new property.

robinmanuelthiel committed:
Revert Style Changes

robinmanuelthiel committed:
Update AccessibiltyDelegete Name in Android

robinmanuelthiel committed:
Update AccessibiltyDelegete Name in Android

robinmanuelthiel committed:
Add UWP Support for Accesiibility

robinmanuelthiel committed:
Update FlexButton.xaml.cs

robinmanuelthiel committed:
Update docs and Nuget

robinmanuelthiel commented:
Released in 0.14.0. Thanks for the meaningful addition!
